### PR TITLE
Clarify the wording about reference builds

### PR DESCRIPTION
**Pull request by sschuberth**

### CODE REVIEW

File: src/main/resources/util/thresholds.properties [hunk 1]
```diff
@@ -21,12 +14,21 @@ description.unstableNew=If the number of new warnings exceeds a threshold then a
...
-description.onlyUseStableBuildsAsReference=The number of new warnings will be calculated based on the last \
-                   stable build, allowing reverts of unstable builds where the number of warnings was decreased. \
-                   Note that the last stable build is evaluated only by inspecting the unit test failures. The \
-                   static analysis results are not considered.
-title.usePreviousBuildAsReference=Use previous build as reference
-description.usePreviousBuildAsReference=If set then the number of new warnings will always be calculated based on the \
-            previous build. Otherwise the reference build (i.e., the build that introduced the new warnings) is used \
-            to calculate the new warnings.
-compute.new.title=Compute new warnings (based on reference build)
+compute.new.title=Compute new warnings (based on a reference build, see below; by default this is the last successful \
```
Comment (uhafner) [inline src/main/resources/util/thresholds.properties]: ; is a comment? At least in github rendering.

File: src/main/resources/util/thresholds.jelly [hunk 2]
```diff
@@ -130,14 +130,14 @@
             </tbody>
           </table>
         </f:entry>
-        <f:entry title="${%Use delta for new warnings}" description="${%description.useDeltaValues}">
```
Comment (uhafner) [inline src/main/resources/util/thresholds.jelly]: this breaks the other properties files (de, fr, ...)

File: src/main/resources/util/thresholds.jelly [hunk 3]
```diff
@@ -130,14 +130,14 @@
...
+        <f:entry title="${%title.useDeltaValues}" description="${%description.useDeltaValues}">
           <f:checkbox name="useDeltaValues" checked="${instance.useDeltaValues}" />
         </f:entry>
         <f:entry title="${%title.usePreviousBuildAsReference}" description="${%description.usePreviousBuildAsReference}">
           <f:checkbox name="usePreviousBuildAsReference"
             checked="${instance.usePreviousBuildAsReference}" />
         </f:entry>
-        <f:entry title="${%Only use stable builds as reference}"
+        <f:entry title="${%title.onlyUseStableBuildsAsReference}"
```
Comment (uhafner) [inline src/main/resources/util/thresholds.jelly]: this breaks the other properties files (de, fr, ...)

File: src/main/resources/util/thresholds.properties [hunk 4]
```diff
@@ -21,12 +14,21 @@ description.unstableNew=If the number of new warnings exceeds a threshold then a
...
+                   build)
+title.useDeltaValues=Use delta for new warnings
+description.useDeltaValues=If set, the number of new warnings is computed by subtracting the total number of warnings \
+                   of the reference build from the total number of warnings of the current build. This may lead to \
+                   wrong results if you have both fixed and new warnings in a build. If unset, the number of new \
+                   warnings is computed by an asymmetric set difference of the warnings in the current build and the \
+                   warnings in the reference build. This will find all new warnings even if the number of total \
+                   warnings has decreased. However, sometimes false positives will be reported due to minor changes in \
+                   a warning (e.g. refactoring of variables or method names).
+title.onlyUseStableBuildsAsReference=Only use stable builds as the reference
```
Comment (uhafner) [inline src/main/resources/util/thresholds.properties]: As short as possible (this is the label and looks ugly if too long).

File: src/main/resources/util/thresholds.properties [hunk 5]
```diff
@@ -21,12 +14,21 @@ description.unstableNew=If the number of new warnings exceeds a threshold then a
...
+description.onlyUseStableBuildsAsReference=Use the last stable build as the reference to compute the number of new \
+                   warnings against. This allows to ignore interim unstable builds for which the number of warnings \
+                   decreased. Note that the last stable build is evaluated only by inspecting the unit test failures. \
+                   The static analysis results are not considered.
+title.usePreviousBuildAsReference=Always use the previous build as the reference
```
Comment (uhafner) [inline src/main/resources/util/thresholds.properties]: As short as possible (this is the label and looks ugly if too long)

File: src/main/resources/util/thresholds.properties (hunk 1 above)
Comment (sschuberth) [inline src/main/resources/util/thresholds.properties]: No, that should have been a literal ";" as part of the title. Do I need to escape that?

File: src/main/resources/util/thresholds.properties [hunk 6]
```diff
@@ -21,12 +14,21 @@ description.unstableNew=If the number of new warnings exceeds a threshold then a
...
-                   stable build, allowing reverts of unstable builds where the number of warnings was decreased. \
-                   Note that the last stable build is evaluated only by inspecting the unit test failures. The \
-                   static analysis results are not considered.
-title.usePreviousBuildAsReference=Use previous build as reference
-description.usePreviousBuildAsReference=If set then the number of new warnings will always be calculated based on the \
-            previous build. Otherwise the reference build (i.e., the build that introduced the new warnings) is used \
-            to calculate the new warnings.
-compute.new.title=Compute new warnings (based on reference build)
+compute.new.title=Compute new warnings (based on a reference build, see below; by default this is the last successful \
+                   build)
```
Comment (uhafner) [inline src/main/resources/util/thresholds.properties]: Also: this is a label and should be as short as possible, maybe we should add an additional help/description that makes the naming of a reference build more clear.

File: src/main/resources/util/thresholds.properties (hunk 1 above)
Comment (uhafner) [inline src/main/resources/util/thresholds.properties]: I don't know, just wondered about the highlighting in GitHub.

File: src/main/resources/util/thresholds.jelly (hunk 2 above)
Comment (sschuberth) [inline src/main/resources/util/thresholds.jelly]: True. I'll simply drop this commit as it's too involving for me to update all translations.

File: src/main/resources/util/thresholds.jelly (hunk 3 above)
Comment (sschuberth) [inline src/main/resources/util/thresholds.jelly]: True. I'll simply drop this commit as it's too involving for me to update all translations.

**Pull request merged**